GUITestProject/Program.cs

```diff
@@ -0,0 +1,173 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using System.Linq;
+using System.IO;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+
+namespace GUITestProject
+{
+    public class Program
+    {
+        private static ILogger<Program> _logger;
+        private static List<(string ServiceName, string Url)> _servicesInfo;
+        private static List<Process> _runningServices = new();
+
+        private static string ServicesPath => Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..\\..\\..\\..\\"));
```

Comment:
AppDomain.CurrentDomain.BaseDirectory, "..\\..\\..\\..\\") - what does it mean?

Reply:
Climb 4 directories up, start in the same place where exe file